server/datastore/mysql/software.go

```diff
@@ -895,3 +915,82 @@ ORDER BY
 	}
 	return hosts, nil
 }
+
+func (ds *Datastore) InsertVulnerabilities(
+	ctx context.Context,
+	vulns []fleet.SoftwareVulnerability,
+	source fleet.VulnerabilitySource,
+) (int64, error) {
```

Comment:
I don't think we need to return the count, looks like it's always either `0` when `err != nil`, or `len(vulns)` when `err == nil`. Also, see https://github.com/fleetdm/fleet/blob/f648949ebefe992ab87fea84ba930c8199c254c2/server/datastore/mysql/mysql.go#L959-L963.